WordPress/WordPressTest/PostCompactCellTests.swift

```diff
@@ -41,7 +41,7 @@ class PostCompactCellTests: XCTestCase {
 
         postCell.configure(with: post)
 
-        XCTAssertEqual(postCell.timestampLabel.text, "just now")
+        XCTAssertEqual(postCell.timestampLabel.text, "now")
```

Comment:
Too easy! 😄 

This brings up an interesting question, one to which I don't have an answer.

- Should this test have used something more down to the implementation level of the library, e.g. asserting it's equal to `valueSetInLibraryForNow`? Or
- It's actually great that we test at a higher level here so if we drop the library we don't need to change the tests (_test behavior not implementation_). Or
- Maybe this is at the wrong abstraction level, only the library should concern itself with testing the type of formatting

Again, I don't have an answer, I don't think there's a _right_ answer anyways, just a set of tradeoffs.

Reply:
> Maybe this is at the wrong abstraction level, only the library should concern itself with testing the type of formatting

Yeah, this is more my preference – option 1 IMHO just test that the code does what the code does. Option 2 is basically what this change is, which is an ok balance of ensuring that we're still testing the text behaviour 🤷. Seems the most in the middle